### ISSUE TITLE
[DOCS] Box Annotation figure and code snippet don't match

### ISSUE THREAD

**Issue by danoscarmike** [bokeh/bokeh#11730]
I've been working through the "first steps" material. In the [Using annotations](https://docs.bokeh.org/en/latest/docs/first_steps/first_steps_3.html#using-annotations) section of First Steps 3, there is a mis-match between the 'Box annotation example' figure and the step by step code snippet. The full finished code section matches the figure, but not the earlier snippet.

The code snippet sets `BoxAnnotation` arguments suggesting a mid-box between 80 and 180. The figure and final code have a mid-box between 20 and 80.

Suggested solution: update the step by step code snippet to reflect the image and final code.

To the maintainers: I'd be delighted to make this contribution myself if you agree it's appropriate/desirable.

**Comment by danoscarmike** [comment]
I've worked up a couple of examples to illustrate the new color combinations. I'm not convinced that green and yellow provide sufficient contrast, when a low alpha is applied. 

![Screen Shot 2021-10-07 at 4 20 47 PM](https://user-images.githubusercontent.com/3130752/136476112-14d5b378-8381-4b7e-b7f9-7d8b9d3c7a96.png)

![Screen Shot 2021-10-07 at 4 29 19 PM](https://user-images.githubusercontent.com/3130752/136476099-79d95e89-4f56-4cf4-86b9-e2046c324069.png)

The second example includes a hatching which gives readers/viewers a second signal to distinguish the areas on the chart. However I recognize this adds a little complexity/convolution to the code which might not be in line with the philosophy of the first steps guide. 

```python
low_box = BoxAnnotation(top=20, fill_alpha=0.1, fill_color="yellow", hatch_pattern=".", hatch_alpha=0.1)
mid_box = BoxAnnotation(bottom=20, top=80, fill_alpha=0.1, fill_color="green")
high_box = BoxAnnotation(bottom=80, fill_alpha=0.1, fill_color="yellow", hatch_pattern=".", hatch_alpha=0.1)
```

By the way, when I build the docs locally and serve (`make clean html serve`) the docs don't render the Bokeh plots. Is this working as intended? Or is there a config setting that I'm missing?

**Comment by bryevdv** [comment]
cc @tcmetzger perhaps we should add a `tag: accessibility` ?

I am fine with either upping the alpha to improve contrast, or adding hatching

**Comment by tcmetzger** [comment]
@danoscarmike Thank you for taking care of this! I'd say we should keep the example as simple as possible, i.e. use a different `fill_alpha` to increase contrast instead of adding a hatch pattern.

When you build docs locally, you should definitely be able to see all the plots in your local docs. One thing to try: Make sure your local BokehJS is up to date (https://docs.bokeh.org/en/latest/docs/dev_guide/setup.html#build-and-install-locally), then run `BOKEH_DOCS_CDN=local make clean html serve` to make sure Sphinx is building the docs with your up-to-date, local version of BokehJS (as described in the "Note" box in https://docs.bokeh.org/en/latest/docs/dev_guide/documentation.html#build-bokeh-s-documentation).
